public/index.html

```diff
@@ -47,7 +47,7 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-    <script type="text/javascript" src="https://cdn.fromdoppler.com/doppler-ui-library/v1.1.1/js/app.js"></script>
+    <script type="text/javascript" src="https://cdn.fromdoppler.com/doppler-ui-library/build308/js/app.js"></script>
```

Comment:
dont forget before merge put the correct version

Reply:
@damianmuti since the merge has been already done in the _UI Library_ you can update it right now.